projects/ccd-case-ui-toolkit/src/lib/shared/components/create-case-filters/create-case-filters.component.html

```diff
@@ -1,27 +1,27 @@
 <form  (ngSubmit)="apply()">
   <div class="form-group">
-    <label class="form-label" for="cc-jurisdiction">Jurisdiction</label>
+    <label class="form-label" for="cc-jurisdiction">{{'Jurisdiction' | rpxTranslate}}</label>
     <select class="form-control ccd-dropdown" id="cc-jurisdiction" name="jurisdiction" [formControl]="filterJurisdictionControl" (change)="onJurisdictionIdChange()">
-      <option value="">--Select a value--</option>
-      <option *ngFor="let j of jurisdictions" [value]="j.id">{{j.name}}</option>
+      <option value="">{{'--Select a value - -' | rpxTranslate}}}}</option>
```

Comment:
suppose we don't translate any signs but only the 'Select a value', and some other issues are '- -', and 4 '}}}}'?
```suggestion
      <option value="">--{{'Select a value' | rpxTranslate}}--</option>
```